### DISCUSSION TITLE
[Devservice] [Database] Can't share databases?

### DISCUSSION THREAD

**Discussion by agreedSkiing** [quarkusio/quarkus#44827] [Q&A] [answered]
Am I missing something or does the .shared flag not exists for database devservice, the generic one?

[a comment by an automated account is not shown]

**Reply by geoand** [reply]
cc @yrodiere

**Comment by yrodiere** [comment, marked as answer]
You're not missing anything; nobody implemented it (yet): https://github.com/quarkusio/quarkus/issues/40376
We accept pull requests :)
